Tests/test_SeqIO.py

```diff
@@ -4422,7 +4422,7 @@ def test_empty_file(self):
         """Check parsers can cope with an empty file."""
         for t_format in SeqIO._FormatToIterator:
             if t_format in SeqIO._BinaryFormats or \
-               t_format in ("uniprot-xml", "pdb-seqres", "pdb-atom", "cif-atom"):
+               t_format in ("uniprot-xml", "pdb-seqres", "pdb-atom", "cif-atom", "cif-seqres"):
```

Comment:
Rather than a simple continue, the test ought to be for the expected exception.

Reply:
True, that makes sense. I see that there are more formats in SeqIO._BinaryFormats ("gck", "seqxml", "snapgene", "nib", "xdna") that currently do not have a consistent error message for empty files. Do we make them raise the same `ValueError("Empty file")` too?

Reply:
That would expected for #1838, yes.